app/browser/api/ledger.js

```diff
@@ -1063,6 +1063,9 @@ const backupKeys = (state, backupAction) => {
 
   if (backupAction === 'print') {
     tabs.create({url: appUrlUtil.aboutUrls.get('about:printkeys')})
+
+    // we do not check whether the user actually printed the backup word list
+    ledgerState.setBackupStatus(state, true)
```

Comment:
We need to assign this call to the state, so that we actually save this change:

`ledgerState.setBackupStatus(state, true)` -> `state = ledgerState.setBackupStatus(state, true)`